CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Internal
 
+- Add nulish coalescence operator in Number widget @iRohitSingh
```

Comment:
I would call this a bugfix, "Fix number widget when the value is 0"

Reply:
fixed